gradient/api_sdk/repositories/notebooks.py

```diff
@@ -14,7 +14,18 @@ class CreateNotebook(GetNotebookApiUrlMixin, CreateResource):
     SERIALIZER_CLS = serializers.NotebookSchema
 
     def get_request_url(self, **kwargs):
-        return "notebooks/createNotebook"
+        return "notebooks/v2/createNotebook"
+    
+    def _process_instance_dict(self, instance_dict):
+        # the API requires this field but marshmallow does not create it if it's value is None
+        instance_dict.setdefault("containerId")
```

Comment:
strange, it should. are other fields serialized correctly?

Reply:
This is more an api side issue, we should make the api accept `containerid` or `containerName` but the api requires containerId in all cases for whatever reason.